agent/tool-scripts/postprocess/samples/prometheus-metrics/metrics-stdout

```diff
@@ -0,0 +1,380 @@
+{"metrics": [{"value": "22"}], "@timestamp": 1491342293, "type": "COUNTER", "name": "admission_quota_controller_adds"}
```

Comment:
So what does the postprocess step for prometheus-metrics generate the same file as output that it reads in?